### ISSUE TITLE
FloatLabel: wrong type

### ISSUE THREAD

**Issue by maitrungduc1410** [primefaces/primereact#6299]
### Describe the bug

when I use FloatLabel, typescript is showing typecheck error though no issue at runtime

```js
<FloatLabel>
  <InputText  />
  <label htmlFor="username">Username</label>
</FloatLabel>
```

<img width="1163" alt="Screenshot 2024-04-04 at 3 07 28 PM" src="https://github.com/primefaces/primereact/assets/16630193/527fb6c4-b77e-4254-91d7-28dd02eefa15">

And in console I also see this error:
<img width="1337" alt="Screenshot 2024-04-04 at 3 09 11 PM" src="https://github.com/primefaces/primereact/assets/16630193/670af8c6-3145-4553-a02a-00ca23581d2a">

### PrimeReact version

10.6.0

### React version

18.x

### Language

TypeScript

### Build / Runtime

Next.js

**Comment by melloware** [comment]
Duplicate of: https://github.com/primefaces/primereact/issues/6268

Fixed for next release.

**Comment by rjahn** [comment]
This problem is not fixed in https://github.com/primefaces/primereact/releases/tag/10.6.3 and it doesn't look like a duplicate of #6268 ?

I use 10.6.3 and have the same error message:

Type '{ children: Element[]; }' has no properties in common with type 'IntrinsicAttributes & FloatLabelProps & RefAttributes<HTMLSpanElement>'.

Can't run the application and can't build it.

**Comment by melloware** [comment]
Dupe of this one fixed for 10.6.4: https://github.com/primefaces/primereact/issues/6387
